packages/playground/website/vite.config.ts

```diff
@@ -134,6 +135,13 @@ export default defineConfig(({ command, mode }) => {
 					}
 				},
 			} as Plugin,
+			/**
+			 * Generate a list of files needed for the website to function offline.
+			 */
+			listAssetsRequiredForOfflineModePlugin({
+				outputFile: 'assets-required-for-offline-mode.json',
+				distDirectoriesToList: ['/', '../remote', '../client'],
```

Comment:
`/` feels icky, as if this was listing the filesystem root. Can we make it `./`?